### DISCUSSION TITLE
ImageWriter for VapourSynth

### DISCUSSION THREAD

**Discussion by Jay123210599** [staxrip/staxrip#1319] [Q&A] [answered]
How do I use ImageWriter with VapourSynth? If that's not possible, is there an alternative tool for it? https://avisynthplus.readthedocs.io/en/latest/avisynthdoc/corefilters/imagewriter.html

**Comment by jessielw** [comment, marked as answer]
https://github.com/vapoursynth/vs-imwri

**Reply by Jay123210599** [reply]
How does that work?

**Reply by Dendraspis** [reply]
You've got a more than detailed answer on Discord, so stop it!
